host/drivers/transport/transport_drv.c

```diff
@@ -503,7 +507,7 @@ esp_err_t send_slave_config(uint8_t host_cap, uint8_t firmware_chip_id,
 	uint16_t len = 0;
 	uint8_t *sendbuf = NULL;
 
-	sendbuf = malloc(512); /*Arbitrary number*/
+	sendbuf = mempool_alloc(((struct mempool*)chan_arr[ESP_PRIV_IF]->memp), 512, true);
```

Comment:
This change does not look correct. It is allocating mempool memory from `char_arr`. `char_arr` is used for another purpose in ESP-Hosted.

Reply:
Yes, it would be better to use g_h.funcs->_h_calloc